src/CosmosDB/CosmosDB/CosmosDBAccount/NewAzCosmosDBDatabaseToRestore.cs

```diff
@@ -0,0 +1,39 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+using Microsoft.Azure.Commands.CosmosDB.Helpers;
+using Microsoft.Azure.Commands.CosmosDB.Models;
+
+namespace Microsoft.Azure.Commands.CosmosDB
+{
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "CosmosDBDatabaseToRestore"), OutputType(typeof(PSDatabaseToRestore))]
+    public class NewAzCosmosDBDatabaseToRestore : AzureCosmosDBCmdletBase
```

Comment:
Said `New-AzCosmosDBRestore` in https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/701, while it's `New-AzCosmosDBDatabaseToRestore` here, please have a look

Reply:
yeah, we updated the verb for Restore. New-AzCosmosDBRestore => Restore-AzCosmosDBAccount
And, the api (New-AzCosmosDBDatabaseToRestore), only creates the list of databases/collections that we need to restore and this is passed as argument to "Restore-AzCosmosDBAccount" api.

Reply:
So what's New-AzCosmosDBRestore for here?

Reply:
We don't have 'New-AzCosmosDBRestore' in stable cmdlets. We have renamed this one from preview version to Restore-AzCosmosDBAccount.